graphqlapi/src/main/java/io/stargate/graphql/schema/cqlfirst/dml/fetchers/QueryFetcher.java

```diff
@@ -84,6 +94,31 @@ public QueryFetcher(
     return result;
   }
 
+  private Map<String, Object> addAggregationResults(
+      Map<String, Object> columns, DataFetchingEnvironment environment, Row row) {
+    List<SelectedField> valuesFields = environment.getSelectionSet().getFields("values");
+    if (valuesFields.isEmpty()) {
+      return columns;
+    }
+    for (SelectedField valuesField : valuesFields) {
+      for (SelectedField selectedField : valuesField.getSelectionSet().getFields()) {
+        if (selectedField.getName().equals(INT_FUNCTION)) {
+          String alias = selectedField.getAlias();
+          // put the returned value as alias
+          if (alias != null) {
+            columns.put(alias, row.getInt(alias));
+          }
+          //
+          else {
+            // if there is no alias, use the value as is.
```

Comment:
I wonder if we should disallow using function calls without aliases. I think it may be reasonable as it offers a lot better UX for end-users than handling a generated function name. On the other hand, I am not sure if this is not too limiting. @olim7t wdyt?